tests/tests.py

```diff
@@ -19,6 +19,8 @@
 from opcua import Event
 from opcua import ObjectIds
 from opcua import AttributeIds
+from opcua.uaprotocol_auto import extensionobject_from_binary
+from opcua.uaprotocol_auto import extensionobject_to_binary
```

Comment:
this is a bit strange why not "from opcua.uaprotocol import extensionobject_from_binary"?